README.md

```diff
@@ -31,12 +33,31 @@ class MainTabBarController: StickyViewControllerSupportingTabBarController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        updateCollapsedHeight(to: 50.0)
-        updateAnimationDuration(to: 0.5)
+        collapsedHeight = 50.0
+        animationDuration = 0.5
     }
 }
 
 ```
+
+By overriding initialisers of the tabbar controller:
+
+```swift
+override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    // here if you are using xib
+    collapsedHeight = 50.0
+    animationDuration = 0.5
+}
+
+required init?(coder: NSCoder) {
+    super.init(coder: coder)
+    // configure also on required init (if you are using storyboard for example)
```

Comment:
Not sure if these comments are accurate enough though. @Thurman1776-

Reply:
I think it is!